examples/with_built_value/pubspec.yaml

```
@@ -3,13 +3,16 @@ publish_to: none
 version: 1.0.0
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=2.14.0 <3.0.0'
 
 dependencies:
   drift: ^2.0.0-dev
   built_value: ^8.0.0
 
 dev_dependencies:
   drift_dev: ^2.0.0-dev
-  built_value_generator: ^8.1.1
+  built_value_generator: ^8.1.3
   build_runner: ^2.0.0
+
+dependency_overrides:
+  analyzer: ^5.0.0
```

Could you add a comment here explaining why the override is necessary?

```suggestion
dependency_overrides:
  # `built_value_generator` doesn't support the latest analyzer yet
  analyzer: ^5.0.0
```